fm-zones.php

```diff
@@ -33,7 +33,7 @@
 
 define( 'FMZ_PATH', dirname( __FILE__ ) );
 define( 'FMZ_URL', trailingslashit( plugins_url( '', __FILE__ ) ) );
-define( 'FMZ_VERSION', '0.1.9' );
+define( 'FMZ_VERSION', '0.2.0' );
```

Comment:
Not sure if the versioning scheme really matters yet, but I personally would probably have gone to `0.1.10` or something like that